code/game/objects/effects/mines.dm

```diff
@@ -129,6 +129,7 @@
 		new /datum/hallucination/delusion(victim, TRUE, "demon",duration,0)
 
 	var/obj/item/twohanded/required/chainsaw/doomslayer/chainsaw = new(victim.loc)
+	add_logs(victim, "has entered a blood frenzy")

```

Comment:
I don't think `add_logs` is the right proc to use here - looks like it's intended for attack logs with both an attacker and a target involved. Maybe just `log_attack` directly if you're sure you want it to go to the attack log?

Reply:
ah thanks. just wanted to have it show somewhere so admins can easily see when the person gained and lost their murderbone privilege from the frenzy.